docs/code_formatting_rules.md

```diff
@@ -203,6 +203,9 @@ func foo(a, b,
 
 func baz(a, b, c) (d,
 	error) {
+
+func longFunctionName(
+	a, b, c (d, error) {
```

Comment:
nit: missing closing parenthesis of input arguments.

Reply:
upps sry 😅